plugins/API/API.php

```diff
@@ -569,7 +569,11 @@ public function getSuggestedValuesForSegment($segmentName, $idSite)
         $suggestedValuesCallbackRequiresTable = false;
 
         if (!empty($segment['suggestedValuesApi']) && is_string($segment['suggestedValuesApi']) && !Rules::isBrowserTriggerEnabled()) {
-            $now = Date::now()->setTimezone(Site::getTimezoneFor($idSite));
+            if ($idSite === 'all') {
+                $now = Date::now()->setTimezone(\Piwik\Plugins\SitesManager\API::getInstance()->getDefaultTimezone());
+            } else {
+                $now = Date::now()->setTimezone(Site::getTimezoneFor($idSite));
+            }
```

Comment:
This makes the `getSuggestedValuesForSegment` API also available for `idSites=all`. Before it was throwing an exception as `Site('all')` isn't possible.